Automattic/wp-calypso
# Add PlanPurchaseFeatures block component

Part for fixing #6758. Add a new block component which will render after plan purchase features such as no ads or Wordads Instant for every WPcom and Jetpack plan. Define plan purchase features their own files and then render these features based on `plan` prop to `PlanPurchaseFeatures` component.

## Testing Instructions

1. Navigate to devdocs -> blocks -> Plan Purchase Features Example;
2. You should see the Personal, Premium, Business and Jetpack plans with their features;
3. To test if the features are working as they should, firstly go to My Sites and select a proper site for testing (ie Business site for testing Business features), then go to the devdocs (without reloading, just navigating within Calypso). This way, `selectedSite` will be set to the proper site;
4. Are the columns nice?
5. Does it look good on smaller devices?

## Screenshot

![selection_032](https://cloud.githubusercontent.com/assets/4988512/17745054/afd45282-64aa-11e6-8520-a069b8232b85.jpg)

## Todo

- [x] Add proper styling: use flex, adjust colors, spacing and paddings
- [x] Add all features to Business and Personal plans
- [x] Add Jetpack plan features
- [x] Make sure all features work in this manner (for accordingly selected site)
- [x] Add header (which plan you are on) and happiness (contact support) features
- [x] Use column alignment instead of row
- [x] Add mobile styles (media queries)
- [x] Split features to their own separate files
- [x] Reorder features according to @apeatling's screenshot

cc @rralian @gwwar @artpi @retrofox @apeatling 

Test live: https://calypso.live/?branch=add/plan-purchase-features
